content/en/docs/use-cases/running-jobs.md

```diff
@@ -0,0 +1,142 @@
+---
+title: Running pre and post-deployment jobs with Flux
+linkTitle: Running Kubernetes Jobs with Flux
+description: "How to run Kubernetes Jobs before and after an application deployment with Flux."
+weight: 10
+---
+
+Additional considerations have to be made when managing Kubernetes Jobs with Flux.
+By default, if you were to have Flux reconcile a Job resource,
+it would apply it once to the cluster, the Job would create a Pod that can either error or run to completion.
+Attempting to update the Job manifest after it has been applied to the cluster is not be allowed, as changes to the
+Jobs `spec.Completions`, `spec.Selector` and `spec.Template` are not permitted by the Kubernetes API.
+To be able to update a Kubernetes Job, the Job has to be recreated by first being
+removed and the reapplied to the cluster.
+
+## Repository structure
+
+A typical use case for running Kubernetes Jobs with Flux is to implement pre-deployment tasks
+for e.g. database scheme migration and post-deployment jobs e.g. cache refresh.
```

Comment:
```suggestion
for e.g. database scheme migration and post-deployment jobs (like cache refresh).
```
omits the repetition of "e.g." that's already used some words prior to this location.